src/services/vanilla.js

```diff
@@ -206,11 +206,12 @@ async function createVanillaGroup (challenge) {
       logger.info(`Creating Vanilla entities for the '${challengeDetailsDiscussion.name}' discussion ....`)
 
       const groupNameTemplate = _.template(groupTemplate.group.name)
+      const shortGroupName = groupNameTemplate.slice(0,config.FORUM_TITLE_LENGTH_LIMIT);
```

Comment:
This will throw an error. `groupNameTemplate` is gonna be a `Function` thus `.slice` will not exist in it.